cmd/agent/main.go

```diff
@@ -11,7 +11,7 @@ import (
 
 func main() {
 	flag.StringVar(&opts.configPath, "config", "/etc/datadog-agent/datadog.yaml", "Path to datadog.yaml config")
-	flag.StringVar(&opts.netConfigPath, "network-config", "/etc/datadog-agent/network-tracer.yaml", "Path to network-tracer.yaml config")
+	flag.StringVar(&opts.netConfigPath, "network-config", "/etc/datadog-agent/network-ebpf.yaml", "Path to network-tracer.yaml config")
```

Comment:
Is this a breaking change that we need to call out?

Reply:
oops good catch! this should not have changed